src/main/java/com/aws/greengrass/clientdevices/auth/certificate/infra/BackgroundCertificateRefresh.java

```diff
@@ -148,6 +148,8 @@ public synchronized void run() {
             return;
         }
 
+        logger.atInfo("Running background task: Refreshing client certificates");
```

Comment:
this will not log anything. `at<x>` needs to be followed by `.log()`. 

Use `logger.info()` to log at info level.